cmd/mungedocs/mungedocs.go

```diff
@@ -28,6 +29,8 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+const latestReleaseBranch = "release-1.1"
```

Comment:
I'm afraid this PR has to be run on the master. Because what this PR does is deciding if the munger should add a link in a doc in the ***master***.
Specifically, the *here* link in line "The latest release of this document can be found [here](http://releases.k8s.io/release-1.1/docs/design/access.md)" in https://github.com/kubernetes/kubernetes/blob/master/docs/design/access.md.